packages/virtual/network/package.mk

```diff
@@ -23,7 +23,15 @@ if [ "$OPENVPN_SUPPORT" = "yes" ]; then
 fi
 
 if [ "$WIREGUARD_SUPPORT" = "yes" ]; then
-  PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET wireguard-tools wireguard-linux-compat"
+  # projects using Linux 5.6+ can use the in-kernel module
+  case $PROJECT in
+    Allwinner|Amlogic|Generic|NXP)
+      PKG_DEPENDS_TARGET="$PKG_DEPENDS_TARGET wireguard-tools"
```

Comment:
What about Rockchip and Qualcomm - aren't they using the default kernel (currently 5.6.y)?

Reply:
I'm not entirely sure what kernels are being used, but the current defconfigs in master show 5.4 (Rockchip) and 5.2 (Qualcomm) so I didn't add them. @Kwiboo @lrusak can you confirm I am right please?

Reply:
@lrusak confirmed Qualcomm is on default kernel, so I added that to the file. @Kwiboo confirmed RK is still in the process of being bumpedd up to 5.6, so until then it will still need the compat module.